### ISSUE TITLE
Pref to turn off JavaScript (iOS)

### ISSUE THREAD

**Issue by brentsimmons** [Ranchero-Software/NetNewsWire#4323]

**Comment by stuartbreckenridge** [comment]
But, but, but...I use Javascript in custom themes...e.g., to make the clock an updating gadget.


https://github.com/user-attachments/assets/ec0efb73-6599-454c-b363-c26c72c38741

**Comment by brentsimmons** [comment]
Don’t worry! It will default to being on — allow JavaScript

**Comment by stuartbreckenridge** [comment]
From a theming perspective I added a "requiresJavascript" bool to the Info.plist on PADD. 

For this ticket, would we be ok to run a check on whether the user's theme is using JavaScript before disabling? If the theme is using JS, display an alert asking if they want to proceed?

**Comment by brentsimmons** [comment]
Good idea to add a `requiresJavascript` bool. For now, let’s hold off on checking and alerting. Not sure how I want to handle that yet.

Note that the Mac app has had this setting for quite a while and, to my knowledge, it hasn’t been confusing regarding themes.
